src/main/java/frc/robot/RobotContainer.java

```diff
@@ -61,6 +70,7 @@ private void configureButtonBindings() {
 
     new Button(nick::getAButton).whenPressed(drivebaseSubsystem::zeroGyroscope);
     new Button(nick::getLeftBumper).whenHeld(new DefenseModeCommand(drivebaseSubsystem));
+    new Button(will::getAButton).whenPressed(new PrintCommand("will a button"));
```

Comment:
Is there a reason for this? I assume a debugging line that slipped in - we use the ``a button`` on the rotation branch.

Reply:
We can just remove this yeah